### ISSUE TITLE
tracking requests fails

### ISSUE THREAD

**Issue by ZjMNZHgG5jMXw** [Microsoft/ApplicationInsights-Go#1]
`telpad`, once tweaked to send request tracks via `telemetryClient.TrackRequest()`, 
- [ ] complains about the missing id in the json payload of the request and, when fixed,
- [ ] the duration format is rejected by the server.

**Comment by ZjMNZHgG5jMXw** [comment]
See #2 for a solution that works for me.

**Comment by ppanyukov** [comment]
I have just had the same with most recent version of this SDK. Here is what we get from app insights server (I have formatted JSON to read easier):

```
Response: 206 -

{
    "errors": [
        {
            "index": 1,
            "message": "109: Field 'id' on type 'RequestData' is required but missing or empty. Expected: string, Actual: ",
            "statusCode": 400
        },
        {
            "index": 3,
            "message": "109: Field 'id' on type 'RequestData' is required but missing or empty. Expected: string, Actual: ",
            "statusCode": 400
        }
    ],
    "itemsAccepted": 3,
    "itemsReceived": 5
}
```

**Comment by jjjordanmsft** [comment]
This ought to be fixed with ba3b5aa
